chcemvediet/apps/inforequests/tests/test_models/test_inforequest_draft.py

```diff
@@ -48,6 +51,15 @@ def test_content_field_default_value_if_omitted(self):
         draft = self._create_inforequest_draft(omit=[u'content'])
         self.assertEqual(draft.content, ())
 
+    def test_modified_field(self):
+        dt = local_datetime_from_local(u'2014-10-05 10:33:00')
+        draft = self._create_inforequest_draft(modified=dt)
+        self.assertAlmostEqual(draft.modified, utc_now(), delta=datetime.timedelta(seconds=10))
+
+    def test_modified_field_default_value_if_omitted(self):
+        draft = self._create_inforequest_draft(omit=[u'modified'])
+        self.assertAlmostEqual(draft.modified, utc_now(), delta=datetime.timedelta(seconds=10))
```

Comment:
`self.assertAlmostEqual` s `timedelta(seconds=10)` je moc vela, ked sa porovnava vzhladom na aktualny cas. Kedze za 10 sekund sa stane obrovske mnozstvo veci pocas testov. Bud zniz 10 sekund na nieco mensie. Alebo sa timewarpni do nejakeho exotickeho casu.

Potom sa pozri, ci aj pri inych testoch, kde robime analogicky assert, to netreba upravit.

Reply:
:ok: